circle.yml

```diff
@@ -9,7 +9,7 @@ machine:
 
 dependencies:
   pre:
-    - pip install awscli
+    - npm install -g npm@3
```

Comment:
`awscli` is used to sync to s3 (future dist assets).  The ta-script package will handle installing it if it doesn't exist.